From bed4dcb1c497557d0a3fc4dbd9fce3b1bde57d93 Mon Sep 17 00:00:00 2001
From: Amarnath Valluri <amarnath.valluri@intel.com>
Date: Tue, 22 Aug 2017 16:35:04 +0300
Subject: [PATCH] fix build failure

utype/utils.h:107:24: error: redefinition of '__u_bswap16'
|  static inline uint16_t __u_bswap16(uint16_t val)
|                         ^~~~~~~~~~~

Signed-off-by: Amarnath Valluri <amarnath.valluri@intel.com>
---
 src/utils.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/utils.h b/src/utils.h
index f9b28fb..a648f10 100644
--- a/src/utils.h
+++ b/src/utils.h
@@ -18,6 +18,9 @@
 
 #pragma once
 
+#ifndef _UTYPE_UTILS_H
+#define _UTYPE_UTILS_H
+
 //#include <sys/types.h>
 #include <stdint.h>
 #include <stdbool.h>
@@ -187,3 +190,4 @@ static inline bool bitfield_test(unsigned long *bits, int bit)
 	return !!(bits[bit / BITS_PER_LONG] & (1UL << (bit % BITS_PER_LONG)));
 }
 
+#endif //_UTYPE_UTILS_H
-- 
2.7.4

